File: client/js/PurchasePeer.js


fan.vaseClient.PurchasePeer = function() {};

fan.vaseClient.PurchasePeer.instance = new fan.vaseClient.PurchasePeer();

fan.vaseClient.PurchasePeer.make = function() {
  return fan.vaseClient.PurchasePeer.instance;
}

fan.vaseClient.PurchasePeer.prototype.start = function(self, name, type)
{
  return false;
}

File: client/java/fan/vaseClient/PurchasePeer.java
package fan.vaseClient;


public class PurchasePeer {

    public static PurchasePeer instance = new PurchasePeer();

    static PurchasePeer make(Purchase self) {
        return instance;
    }
    boolean start(Purchase self, String name, long type) {
        self.finish(name, 0);
        return false;
    }
}
